src/fragments/lib/auth/android/signin_next_steps/10_signin.mdx

```diff
@@ -170,22 +175,19 @@ try {
             // flow, and once reset password flow completes, invoke
             // `signIn` api to trigger signIn flow again.
         }
-        AuthSignInStep.CONFIRM_SIGN_UP -> {
-            Log.i("AuthQuickstart", "Confirm signup, additional info: ${nextStep.additionalInfo}")
-            // User was not confirmed during the signup process.
-            // Invoke `confirmSignUp` api to confirm the user if
-            // they have the confirmation code. If they do not have the
-            // confirmation code, invoke `resendSignUpCode` to send the
-            // code again.
-            // After the user is confirmed, invoke the `signIn` api again.
-        }
         AuthSignInStep.DONE -> {
             Log.i("AuthQuickstart", "SignIn complete")
             // User has successfully signed in to the app
         }
     }
 } catch (error: Exception) {
-    Log.e("AuthQuickstart", "Unexpected error occurred: $error")
+   if (error is UserNotConfirmedException) {
+      Log.e("AuthQuickstart", "Failed to sign in due to user not being confirmed", error)
+      displayLog(error.toString(), true)
+      }
```

Comment:
Else should be on the same line as the closing if brace and incorrect indentation here.